x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action.ts

```diff
@@ -2071,63 +2059,6 @@ export default ({ getService }: FtrProviderContext): void => {
               expect(prebuiltRule.version).toBe(readRule.version);
             });
           });
-
-          // if rule action is applied together with another edit action, that can't be applied to prebuilt rule (for example: tags action)
-          // bulk edit request should return error
-          it(`should return error if one of edit action is not eligible for prebuilt rule`, async () => {
```

Comment:
Similarly, this should go to `x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/prebuilt_rule_customization/customization_disabled/customization_via_bulk_editing.ts`.